platform/security/platform-security-core-api/src/main/java/ddf/security/SecurityConstants.java

```diff
@@ -44,7 +44,7 @@
   public static final String SAML_COOKIE_NAME = "org.codice.websso.saml.token";
 
   /** Property key to obtain the legacy saml cookie reference from an incoming HTTP request */
-  public static final String SAML_COOKIE_REF = "or.codice.websso.saml.ref";
+  public static final String SAML_COOKIE_REF = "org.codice.websso.saml.ref";
```

Comment:
❓ Is this actually fixing a long-term problem or will it not break the logic to get legacy saml cookies?

@stustison, do you know?